tools/harness-simulation/README.md

```diff
@@ -10,15 +10,33 @@ Platform developers should modify the THCI implementation and/or the SI implemen
 
 ## POSIX Environment Setup
 
-1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. For example, run the following command in the top directory of OpenThread.
+1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. Taking running OpenThread 1.2 test cases as an example, run the following command in the top directory of OpenThread.
```

Comment:
Taking running OpenThread 1.2 test cases as an example, run the following command in the top directory of OpenThread.

update to

For example, to run OpenThread 1.2 test cases, run the following command in the top directory of OpenThread: